.vib/odoo/cypress/cypress/integration/odoo_spec.js

```diff
@@ -0,0 +1,83 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+import 'cypress-real-events/support';
+
+it('allows creating a company', () => {
+  cy.login();
+  cy.get('[title="Home Menu"]').click();
+  cy.contains('Settings').click();
+  cy.contains('Users & Companies').click();
```

Comment:
Could we avoid some of this steps by using `cy.visit()` or is it needed to navigate through clicks?

Reply:
The URLs are really awkward and long so I would prefer to leave it as it is, please check here: 

![Screenshot 2022-07-22 at 12 24 41](https://user-images.githubusercontent.com/47592680/180419973-42891e48-a4e6-44ed-b85e-42c41e1ea3ad.png)